test/e2e/framework/workload_manager.go

```diff
@@ -138,7 +138,12 @@ func (wm *WorkloadManager) TrafficManagerProfile() fleetnetv1alpha1.TrafficManag
 		},
 		Spec: fleetnetv1alpha1.TrafficManagerProfileSpec{
 			MonitorConfig: &fleetnetv1alpha1.MonitorConfig{
-				Port: ptr.To(int64(80)),
+				Port:                      ptr.To(int64(80)),
```

Comment:
Why do we need to set the additional fields?

Reply:
to make sure our controller will set these fields